fuse/data/visualizer/visualizer.py

```diff
@@ -0,0 +1,28 @@
+import os
+import matplotlib.pyplot as plt
+from typing import List
+from fuseimg.utils.visualization.multi_label import *
```

Comment:
fuse.data can't import fuseimg.
You should move this code to fuseimg.

Reply:
sorry, this file should be removed, I accidently copied it here